src/Perpetuum/Perpetuum.csproj

```diff
@@ -78,7 +78,6 @@
       <HintPath>..\..\packages\System.Collections.Immutable.1.4.0\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
     </Reference>
     <Reference Include="System.Core">
-      <RequiredTargetFramework>3.5</RequiredTargetFramework>
```

Comment:
Whats this removed for?  Is it safe to do so?

Reply:
We’re targeting .NET 4.6.1, unless we suddenly decide to target .NET 2.0 it’s fine. This gives you an IDE warning if you decide to do such a thing but is largely redundant since VS 2019